shell/components/form/ResourceTabs/index.vue

```diff
@@ -85,13 +85,13 @@ export default {
         STEVE_EVENT_LAST_SEEN,
         STEVE_EVENT_TYPE,
         REASON,
-        headerFromSchemaColString('Subobject', eventSchema, this.$store.getters, true),
-        headerFromSchemaColString('Source', eventSchema, this.$store.getters, true),
+        eventSchema ? headerFromSchemaColString('Subobject', eventSchema, this.$store.getters, true) : null,
+        eventSchema ? headerFromSchemaColString('Source', eventSchema, this.$store.getters, true) : null,
         MESSAGE,
-        headerFromSchemaColString('First Seen', eventSchema, this.$store.getters, true),
-        headerFromSchemaColString('Count', eventSchema, this.$store.getters, true),
+        eventSchema ? headerFromSchemaColString('First Seen', eventSchema, this.$store.getters, true) : null,
+        eventSchema ? headerFromSchemaColString('Count', eventSchema, this.$store.getters, true) : null,
         STEVE_NAME_COL,
-      ]
+      ].filter((header) => header)
```

Comment:
Apparently users may not have access to the events. I noticed this when testing against a user that didn't have access to the local cluster. Let me know if you'd like to resolve this in another way.